python/tvm/meta_schedule/logging.py

```diff
@@ -129,7 +133,7 @@ def create_loggers(
             "class": "logging.FileHandler",
             "filename": "{log_dir}/" + __name__ + ".task_scheduler.log",
             "mode": "a",
-            "level": "INFO",
+            "level": "DEBUG",
```

Comment:
I agree it can be changed to `DEBUG`, but to clarify, all the use of `DEBUG` here are only for the support of outputing different text to file and console, for example, output interactive table to notebook and text table to log file. This does not impact per task logging and that's why this `INFO` level is kept there. Given they are not outputed to screen, I think change to `DEBUG` level still make sense, will do.

Reply:
Yeah I was just not sure about the difference between loggers and handlers and asked the question. Thanks for clarifying!